completed/camunda-8.js

```diff
@@ -2,18 +2,19 @@ import axios from "axios";
 import { getAccessToken } from "./auth.js";
 
 const authorizationConfiguration = {
-  clientId: process.env.ZEEBE_CLIENT_ID,
-  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
-  audience: process.env.ZEEBE_AUDIENCE
+  clientId: process.env.CAMUNDA_CLIENT_ID,
+  clientSecret: process.env.CAMUNDA_CLIENT_SECRET,
+    // These settings come from your .env file. Note that CAMUNDA_TOKEN_AUDIENCE is represented by ZEEBE_TOKEN_AUDIENCE.
+  audience: process.env.CAMUNDA_TOKEN_AUDIENCE
```

Comment:
I think you meant to also change the `ZEEBE_TOKEN_AUDIENCE` in the .env.example, but this name does not align with what's in there.